### PR TITLE
[transformation] Multiple Target Assignment

### PR DESCRIPTION
**Pull request by artbobrov**

### CODE REVIEW

File: src/test/kotlin/org/jetbrains/research/ml/ast/transformations/util/TransformationsTest.kt [hunk 1]
```diff
@@ -42,12 +52,20 @@ open class TransformationsTest(testDataRoot: String) : ParametrizedBaseTest(test
     ) {
         LOG.info("The current input file is: ${inFile.path}")
         LOG.info("The current output file is: ${outFile.path}")
-        val expectedSrc = outFile.content
-        LOG.info("The expected code is:\n$expectedSrc")
         val psiInFile = myFixture.configureByFile(inFile.name)
+        val expectedPsiInFile = myFixture.configureByFile(outFile.name)
+        WriteCommandAction.runWriteCommandAction(project) { // reformat the expected file
+            codeStyleManager.reformat(expectedPsiInFile)
+        }
```
Comment (nbirillo) [inline src/test/kotlin/org/jetbrains/research/ml/ast/transformations/util/TransformationsTest.kt:59]: you use this fragment of code twice. Let's refactor a little bit:
```
protected fun getPsiFile(file: File, toReformatFile: Bolean = true): PsiFile {
        val psiFile = myFixture.configureByFile(file.path)
       if (toReformatFile) {
              WriteCommandAction.runWriteCommandAction(project) { // reformat the expected file
                   codeStyleManager.reformat(psiFile)
               }
       }
        return psiFile
    }
```

```
        ...
        val psiInFile = getPsiFile(inFile)
        val expectedPsiInFile = getPsiFile(outFile)
        ...
```

Moreover, you can call the `getPsiFile` method in the `PsiTreeConverterTest` class:

https://github.com/nbirillo/ast-transformations/blob/master/src/test/kotlin/org/jetbrains/research/ml/ast/gumtree/tree/PsiTreeConverterTest.kt#L56

Here, we get the relative path maybe not the best way. We can delete it and use `file.path`

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great job, but I have a small comment

File: src/test/resources/org/jetbrains/research/ml/ast/transformations/multipleTargetAssignment/data/in_2.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+def main(x):
+    a = 4
+    b = 4
+    c = None
```
Comment (nbirillo) [inline src/test/resources/org/jetbrains/research/ml/ast/transformations/multipleTargetAssignment/data/in_2.py:4]: What shows this test?

Comment (artbobrov) [reply src/test/resources/org/jetbrains/research/ml/ast/transformations/multipleTargetAssignment/data/in_2.py:4]: Not to target simple expressions.

**Review by nbirillo — APPROVED** [review]
Great!

**Review by elena-lyulina — APPROVED** [review]
Cool!

**Pull request merged**